playbooks/generic-collect-logs.yml

```diff
@@ -0,0 +1,61 @@
+---
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- hosts: soc-deployer
+  gather_facts: false
+  vars:
+    # this is where on the target machine the logs will be copied
+    logs_dir: "/tmp/socok8s_gather_logs"
+    zuul:
```

Comment:
(nit) This is being ran in our local Jenkins right? Why are you structuring the vars around zuul? I get the upstream gathering might use this structure, but we could still just define the path.

Reply:
yeah so this is because we are using the upstream playbooks for the gathering, and those are used on zuul primary they all use the ansible vars `{{ zuul.executor.log_root }}` for the logs (see here[0]).

As we just want to leverage those upstream playbooks with as little changes as possible, its easier to just pass those vars from our include_role so the logs get copied in the proper dir.

Im not sure how would we be able to override those vars any other way btw, I just did a quick job, maybe there is a way of overriding those but not that I know of :)


[0] https://github.com/openstack/openstack-helm-infra/blob/master/roles/gather-pod-logs/tasks/main.yaml#L59